plugins/conversion.js

```diff
@@ -0,0 +1,99 @@
+const _ = require('lodash');
+
+module.exports = (function() {
+
+	let bot;
+
+	function doConversion (message, channel) {
+
+		let [command, ...args] = message.parts;
+		let measurement, unit;
+
+		if(!args.length){
+			channel.send('Please provide something to convert.');
+			return true
+		}
+
+		//The user put a space between the measurement and the units
+		if(args.length > 1){
+			measurement = args[0];
+			unit = args[1].toLowerCase();
+		} else {
+
+			//Extract the measurement and units from a single string
+			let measurements = args[0].match(/^[\d.-]+/g);
+			let units = args[0].match(/[^\d.-]+/g);
```

Comment:
You mean measurements/units vs measurement/unit? Not sure what else to call them since the plural is an array of items found by the regex and the singular is the item we'll be operating on.  I'm open to different naming ideas though.